controlplane/kubeadm/internal/workload_cluster_etcd.go

```diff
@@ -93,6 +96,10 @@ func (w *Workload) EtcdIsHealthy(ctx context.Context) (HealthCheckResult, error)
 		}
 
 		member := etcdutil.MemberForName(members, name)
+		if member == nil {
+			response[name] = errors.Wrap(err, "node connected missing from etcd members")
```

Comment:
I find this error message hard to understand. Could we make this a little bit more user friendly

Reply:
`failed to find the member for the machine` ?